src/base/utils.js

```diff
@@ -40,7 +40,7 @@ export function formatTime(time) {
       out += days + ":"
       if (hours < 1) out += "00:"
     }
-    if (hours && hours > 0) out += ("0" + hours).slice(-2) + ":"
+    if (hours && hours > 0 || paddedHours !== undefined && paddedHours) out += ("0" + hours).slice(-2) + ":"
```

Comment:
In javascript `null`, `undefined` and `false` are all falsey so if it's `undefined` it's still ok.